app/models/manageiq/providers/google/cloud_manager/refresh_parser.rb

```diff
@@ -77,11 +77,13 @@ def get_volumes
       def get_snapshots
         snapshots = @connection.snapshots.all
         process_collection(snapshots, :cloud_volume_snapshots) { |snapshot| parse_snapshot(snapshot) }
+        # Also pass the snapshots as templates
+        process_collection(snapshots, :vms) { |snapshot| parse_storage_as_template(snapshot) }
```

Comment:
@gmcculloug @bdunne Do you know if there is a better way to allow provisioning from snapshots than duplicating all snapshots also as templates?